project-generator/src/main/java/io/syndesis/project/converter/DefaultProjectGenerator.java

```diff
@@ -104,9 +123,31 @@ public DefaultProjectGenerator(ConnectorCatalog connectorCatalog, ProjectGenerat
         });
 
         Map<String, byte[]> contents = new HashMap<>();
+
+        for (Templates.Resource additionalResource : generatorProperties.getTemplates().getAdditionalResources()) {
+            String overridePath = generatorProperties.getTemplates().getOverridePath();
+            URL resource = null;
+
+            if (!Strings.isEmpty(overridePath)) {
+                resource = getClass().getResource("templates/" + overridePath + "/" + additionalResource.getSource());
```

Comment:
The logic is that it checks first if the listed resource is part of the "overridden" resources then it search in the default path so the exception is thrown only if the resource is not found in any of the known path, may be we should log a debug level which path is used ?